template.yaml

```diff
@@ -330,19 +330,13 @@ Resources:
       ContainerDefinitions:
         - Essential: true
           Image: CONTAINER-IMAGE-PLACEHOLDER
-          Name: app
           Environment:
             - Name: API_BASE_URL
               Value: !FindInMap [EnvironmentVariables, !Ref Environment, APIBASEURL ]
             - Name: EXTERNAL_WEBSITE_HOST
```

Comment:
we can remove EXTERNAL_WEBSITE_HOST env var as it isn't being used in the FE code

Reply:
EXTERNAL_WEBSITE_HOST env var has already been removed. i just needed renaming. done